src/en/installconfig-upgrade.md

```diff
@@ -0,0 +1,45 @@
+Title: Upgrade from MAAS 1.9
+
+# Upgrade from MAAS 1.9
+
```

Comment:
Good idea about the test environment. I'll add that. I was uncertain about backups as it seems so obvious, but there's no harm in adding it.